scripts/bfb-install

```diff
@@ -459,6 +461,17 @@ cleanup() {
       run_cmd remote "rm -f $RSHIM_PIPE"
     fi
   fi
+
+  if [ $runtime -eq 1 ]; then
+    # Reset to default state.
+    echo "BOOT_RESET_SKIP 0" > ${rshim_node}/misc
+
+    # Cleanup SP2.BIT2 and trigger SWINT2.
+    sp2=`${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 | awk '{print $3}'`
+    sp2=$((sp & ~4))
+    ${BF_REG} $(basename ${rshim_node}) 0x13000c48.64 $sp2 >/dev/null
+    ${BF_REG} $(basename ${rshim_node}) 0x13000318.64 0x4 >/dev/null
+  fi
```

Comment:
Same comment as the first one. It applies to the newly added variable $runtime, $BF_REG etc. We can remove the two function comment lines.

Reply:
done